packages/uauth/src/types.ts

```diff
@@ -0,0 +1,32 @@
+export type UauthInitOptions = {
+  /**
```

Comment:
@V-Shadbolt I added these note block to allow for IDE TS feedback that helps devs..just an fyi from a 🤓 
<img width="689" alt="Screen Shot 2022-11-17 at 10 27 36 AM" src="https://user-images.githubusercontent.com/24457880/202515912-494d1db9-60dc-4ba9-b23c-e0001dfabada.png">